tests/test_telemetry.py

```diff
@@ -485,3 +485,30 @@ def test_context_contains_os():
     context.pop("os")
 
     assert "os" in telemetry._default_context_fields()
+
+
+def test_context_contains_license_hash(monkeypatch: MonkeyPatch):
```

Comment:
We should have the type hint for test return value (-> None) in both test cases.

Reply:
Added 👍